client/hotrod-client/src/main/java/org/infinispan/client/hotrod/multimap/MultimapCacheManager.java

```diff
@@ -12,5 +12,17 @@
     * @return null if no configuration exists as per rules set above, otherwise returns a multimap cache instance
     * identified by cacheName
     */
-   RemoteMultimapCache<K, V> get(String name);
+   default RemoteMultimapCache<K, V> get(String name) {
```

Comment:
We should document this method as returning a multimap that doesn't support duplicates.